src/gui/downloadfromurldialog.cpp

```diff
@@ -28,6 +28,8 @@
 
 #include "downloadfromurldialog.h"
 
+#include <libtorrent/version.hpp>
```

Comment:
Is it needed to something?

Reply:
Look at line 53/55. There is `LIBTORRENT_VERSION_NUM`. In QtCreator it wasn't recognized without the header.

Reply:
@sledgehammer999 
Maybe revert this and  leave it to PR #15297, it will be handled properly.

Reply:
done